students/YingGuo/lession07/html_render.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element():
+    tag_name = "html"
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.content = []
+        if content is not None:
```

Comment:
Minor: This line should be just `else`.

Better yet, the whole `if`-`else` block better be simply `self.content = [] if content is None else [content]`.